readme.md

```diff
@@ -43,7 +43,7 @@ Where we're picking up the app, it has only a functioning index route that uses
 
 Also, it doesn't use `$locationProvider` to remove the hashmark from URLs: all the URLs will be something like index.html#/grumbles/32. This makes things a bit easier for development: without the hashmark refreshing the page doesn't work, but it does with the hashmark.
 
```

Comment:
i've removed all references to `index.html` in the url? not sure why they need it. Going to `localhost:8080/#/grumbles/32` works for me...